### PR TITLE
Accept non-interactive apt-get install commands

### PR DESCRIPTION
**Pull request by comzeradd**
Not sure how this used to work so far (maybe `apt-get` accepts after a time-out period?), but currently our CI test pipelines fail because of `apt-get` waiting for an approval.

### CODE REVIEW

**Comment by comzeradd** [comment]
Ah actually it's because `apt-get` doesn't wait for confirmation if there is only one package (the one requested) that is going to be installed. That changed for `pconv`. It used to install just one package (itself) but now it requires 5, so it needs an approval.

```
The following NEW packages will be installed:
  php-pcov php7.1-pcov php7.2-pcov php7.3-pcov php7.4-pcov
0 upgraded, 5 newly installed, 0 to remove and 4 not upgraded.
```

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**
